docs/typescript.md

```diff
@@ -309,10 +318,10 @@ If you see outdated TypeScript types, you can help update them by contributing t
 ### How to Generate Types
 
 1. Fork [DefinitelyTyped/DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped)
-1. Run the types generator by following the instructions at [motemen/dts-google-apps-script](https://github.com/motemen/dts-google-apps-script)
-1. Copy the type files to your fork of `DefinitelyTyped/DefinitelyTyped`.
-1. View the diff (`git diff`), and make sure that the types look OK.
-1. Make a PR and ask [@grant](https://github.com/grant) for review.
+2. Run the types generator by following the instructions at [motemen/dts-google-apps-script](https://github.com/motemen/dts-google-apps-script)
```

Comment:
Don't change this. Please revert.

Reply:
Not sure what is the cause of this change... grr